api/src/util/env_config.py

```diff
@@ -13,3 +14,8 @@
 
 class PydanticBaseEnvConfig(BaseSettings):
     model_config = SettingsConfigDict(env_file=env_file)
+    environment: str = Field(alias="ENVIRONMENT")
+
+    @property
+    def is_prod(self) -> bool:
+        return self.environment == "prod"
```

Comment:
Don't add anything to this class, this is a base class that about 20 other things are derived from and you'd be affecting all of them
```suggestion

```